app/views/requests/sponsor_request.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1 id="dialogTitle">Share with your proxy group?</h1>
```

Comment:
There is a div that wraps the other action's views that will show up in a dialog (while the home page and the admin pages get full-width).

It's possible that we need to do something more clever with layouts, but in the meantime this page looks vastly different from the other pages.  With the addition of some wrapper divs and making the buttons 12 column we can get a more consistent display.

### Current
![sponsor-form-before](https://cloud.githubusercontent.com/assets/96776/8091642/dcb05388-0f69-11e5-9ef6-3451c40d86be.png)

### Modified
![sponsor-form-after](https://cloud.githubusercontent.com/assets/96776/8091641/dcace55e-0f69-11e5-9cc4-a4814fe1a3c2.png)


```ruby
<div class='<%= dialog_column_class %>'>
  <h1 id="dialogTitle">Share with your proxy group?</h1>

  <%= render 'form_flashes' %>
  <%= render 'searchworks_item_information' %>
  <div class="<%= "#{label_column_offset_class} #{content_column_class}" %>">
    <%= bootstrap_form_for(current_request) do |f| %>
      <%= hash_to_hidden_fields(request_params) %>

      <h2><%= render_user_information %></h2>
      <%= f.form_group help: 'Anyone in your group can check the status of this request and pick it up when it’s available.' do %>
        <%= f.button 'Yes, share with my group.', class: 'btn btn-default col-md-12', value: 'true', name: 'proxy' %>
      <% end %>

      <%= f.form_group help: 'Only you can see and pick up this request.' do %>
        <%= f.button 'No, just me.', class: 'btn btn-cardinal col-md-12', value: 'false', name: 'proxy' %>
      <% end %>
    <% end %>
  </div>

</div>

```